examples/amp-story/amp-story-shopping.html

```diff
@@ -251,6 +243,18 @@
         </script>
       </amp-story-shopping-attachment>
       </amp-story-page>
+       <!--
+          Example of:
+          CTA Button not rendering due to no page attachment.
```

Comment:
Please change "no page attachment" to "no shopping attachment".

Page attachment refers to a different component so we need to be sure to be specific that this is a shopping attachment.
Please update this in all html documents.